package.json

```diff
@@ -38,8 +38,8 @@
     "@stream-io/flat-list-mvcp": "0.10.3",
     "@tsconfig/react-native": "3.0.2",
     "base-64": "1.0.0",
-    "commonmark": "npm:@mattermost/commonmark@0.30.1-0",
-    "commonmark-react-renderer": "github:mattermost/commonmark-react-renderer#2c660491041f7595f6ce5a05f6dc2e30ca769d3a",
+    "commonmark": "github:mattermost/commonmark.js#9175135271b44f5fd9d15b7decebfb79032988eb",
```

Comment:
I forgot that we published that to NPM before. I'm not able to find the exact version we published which is kind of annoying for matching it back to the repo. I don't think it's critical to have on NPM since the git dependency issues are gone with the newer version of NPM and we'd just be publishing it for our own use, but I can do that if you want

Reply:
I think we ended up publishing cause using the `github` reference was causing issues with the SHA hash

Reply:
Right. Looking back at the commit logs, I'm the one that actually published it too 😅

I'll switch that to avoid causing this problem again. At least those hashes won't cause any more problems next time we upgrade Node/NPM for the mobile app since NPM 9 doesn't do those SHA checks for git dependencies any more.

Reply:
I found out where that commit was. I had started to work on shipping commonmark.js, and I published from my local branch, but I never actually pushed it to GitHub.

Now that that's fixed, and the tag is on GitHub, I'll tag a new release, publish it, and use it here